### PR TITLE
[WebProfilerBundle] Fix JS error when toolbar is reloaded

### LINKED ISSUE #44142: JS TypeError when replacing the debug toolbar
### Symfony version(s) affected

5.4.0-BETA3

### Description

When replacing the debug toolbar after an AJAX call using the `Symfony-Debug-Toolbar-Replace` header, a `TypeError` is logged to the console.

```
Uncaught TypeError: can't access property "classList", el is null
    removeClass https://localhost:8000/:22
    showToolbar https://localhost:8000/:22
    initToolbar https://localhost:8000/:22
    loadToolbar https://localhost:8000/:22
    load https://localhost:8000/:22
    onreadystatechange https://localhost:8000/:22
    request https://localhost:8000/:22
    load https://localhost:8000/:22
    loadToolbar https://localhost:8000/:22
    finishAjaxRequest https://localhost:8000/:22
    fetch https://localhost:8000/:22
    promise callback*window.fetch https://localhost:8000/:22
    <anonymous> https://localhost:8000/:16
    setTimeout handler* https://localhost:8000/:15
localhost:8000:22:344
```

### How to reproduce

tl;dr: Create a controller that emits the `Symfony-Debug-Toolbar-Replace` header and perform an AJAX call to that controller.

Long version: use my reproducer that I've published on https://github.com/derrabus/wdt-reproducer

```php
composer install
symfony serve
```

Then access the `/ ` route of the app.

### Possible Solution

_No response_

### Additional Context

The problem is gone if I downgrade only `symfony/web-profiler-bundle` to 5.3.

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix #44142
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
Thanks, that patch fixes the main problem already. But when the app fires multiple concurrent AJAX requests, I get a new error now:

```
Uncaught TypeError: can't access property "id", sfwdt is null
    loadToolbar https://localhost:8000/:31
    load https://localhost:8000/:31
    onreadystatechange https://localhost:8000/:31
    request https://localhost:8000/:31
    load https://localhost:8000/:31
    loadToolbar https://localhost:8000/:31
    finishAjaxRequest https://localhost:8000/:31
    fetch https://localhost:8000/:31
    promise callback*window.fetch https://localhost:8000/:31
    fetchContent https://localhost:8000/:16
    setTimeout handler* https://localhost:8000/:26
localhost:8000:31:22660
```

I've pushed a commit to the reproducer repository that reproduces this problem.

**Review by derrabus — APPROVED** [review]

**Comment by derrabus** [comment]
The race condition is gone as well now. Thank you very much!

**Comment by derrabus** [comment]
Thank you Jérémy.
